log4j-api-kotlin/src/main/kotlin/org/apache/logging/log4j/kotlin/KotlinLogger.kt

```diff
@@ -117,19 +117,19 @@ class KotlinLogger(val delegate: ExtendedLogger) {
     delegate.logIfEnabled(FQCN, level, null, msg, t)
   }
 
-  fun log(level: Level, supplier: () -> Any?) {
+  inline fun log(level: Level, supplier: () -> Any?) {
```

Comment:
Does this affect the caller information in any breaking way?

Reply:
My understanding is that this is both source and binary compatible. I can't say that with 100% certainty but I'll ask on the Kotlin Slack.

Reply:
Yes, confirmed that it is.

Reply:
I mean in the caller location pattern. If you have a pattern layout with `%C`, for example, you can get the name of the caller class with line numbers and things like that. That feature relies upon knowing which stack frames can be safely skipped to find the caller of the logger. By inlining things, you're changing the stack, so you should verify that caller info/method info still work.

Reply:
@jvz No, the caller location seems to work the same as before -- the trace extension still doesn't do the right thing, but that was already a known issue.